api/README.md

```diff
@@ -144,3 +143,36 @@ The list of available handlebars variables:
   - `name`: The service name which is equivalent to the network alias
   - `port`: The exposed port of the service
   - `type`: The type of service. For example, `instance`, `replica`, `app-companion`, or `service-companion`.
+
+## Hooks
+
+Hooks can be used to manipulate the deployment before handing it over to actual infrastructure. For example, based on the app name you can decide to reconfigure your services to use a different DBMS so that you are able to verify that your service works with different DBMSs.
+
+Technically, hooks are Javascript files that provide functions to modify the service configurations. For example, add following section to your PREvant configuration. This configuration snippet enables the _deployemnt hook_ that will be used to modify the services' configurations.
+
+```toml
+[hooks]
+deployment = 'path/to/hook.js'
+```
+
+The hook at `path/to/hook.js` must provide following Javascript function:
+
+```javascript
+function deploymentHook(appName, serviceConfigs) {
+  return serviceConfigs.filter( _ => true );
+}
+```
+
+Note: the templating has been already applied and that the hook has only access to the final value and it cannot modify the original values. 
```

Comment:
something is wrong in this sentence. Did you mean `.. and the hook has ..` or otherwise start it with `Note that the..`

Reply:
yes, you are right.